### DISCUSSION TITLE
Checkout Pro: Entering multiple items doesn't describe the list of items in the checkout

### DISCUSSION THREAD

**Discussion by rafaelfaria** [mercadopago/sdk-js#61] [Q&A] [answered]
So when creating the preference,  I entered multiple items. I specified the title, image, etc. However, the checkout only shows `Produto` and the sum in front of it.

What is the point of specifying multiple products and showing them this way?  
Is there any way we can show the proper list of all the items described that we submitted? This is really important for us.

Please let me know.
Thanks

**Comment by gdeandradero** [comment, marked as answer]
Hello how are you?
Unfortunately, we don't have this feature in CHO PRO, but it is possible that it will be implemented in the future.
You can try using CHO API for a more personalized experience.

If you need help or have any questions, feel free to ask.
Thank you for the contact!!

**Reply by rafaelfaria** [reply]
Thanks, @gdeandradero 

It surprises me that something so basic is not implemented. What is the point of specifying the line items if they not gonna show them? 

Also, is there any ETA on when this feature will be integrated? We decided to go with Checkout Pro because of its simplicity in implementing it, the trade-off is hard to justify since going in the other direction will not only take a longer time to implement, but maintainability also becomes more complex.
It's a shame that you might lose a customer for something so basic.

Thanks

**Reply by gdeandradero** [reply]
Hello, @rafaelfaria 
We forwarded this demand to the responsible team, but we don't have a date for it because has a feature prioritization process.
For more information/questions, we recommend contacting support.
